libraries/chain/controller.cpp

```diff
@@ -2216,8 +2217,7 @@ struct controller_impl {
                   section.read_row(legacy_global_properties, db);
 
                   db.create<global_property_object>([&legacy_global_properties,&gs_chain_id](auto& gpo ){
-                     gpo.initialize_from(legacy_global_properties, gs_chain_id, kv_database_config{},
-                                        genesis_state::default_initial_wasm_configuration);
+                     gpo.initialize_from(legacy_global_properties, gs_chain_id);
```

Comment:
What about genesis_state::default_initial_wasm_configuration ?

Reply:
It is now filled in by `initialize_from()` for simplication.

Reply:
Thanks. Hard to see on phone. I'll review tomorrow.

Reply:
Thank you and appreciate it! You don't need to review the PR on your vacation. I put it up for making sure all CICD pass before you and Greg look it next Monday.